Jenkinsfile.integration

```diff
@@ -87,6 +88,7 @@ pipeline {
                 anyOf {
                     expression { return DocsChanged }
                     expression { return ToxChanged }
+                    expression { return DocsOnlyRun }
```

Comment:
(nit) this is redundant (due to DocsChanged), but not a problem

Reply:
removed